doc_admin/AdminManual.adoc

```diff
@@ -92,30 +95,30 @@ include::./oh_software_compatibility_matrix.adoc[]
 [#jre]
 ==== 3.1.2 Java Runtime Environment (JRE)
 
-Depending on the operating system, there are different ways to install a Java Virtual Machine and the Java Runtime Environment (JRE)
-
-For instance, on a Microsoft Windows® system, the JRE is downloaded and setup file is executed, while on Linux based systems a single command is typed into a terminal window.
+Depending on the operating system, there are different ways to install a Java Virtual Machine and the Java Runtime Environment (JRE).
+The oh.sh / oh.bat set of scripts can download and install the latest JRE available by downloading it from internet; it is also possibile to install it
```

Comment:
Too many `it`s in the sentence; how about instead of `downloading it from internet; it is also possibile to install it` to `downloading it from internet; it is also possible to install the JRE`?  Also note the spelling fix for `possibile`.

Reply:
Rephrased the sentence